tests/llmobs/test_llmobs_service.py

```diff
@@ -1470,6 +1481,37 @@ def test_llmobs_fork_create_span(monkeypatch):
         llmobs_service.disable()
 
 
+def test_llmobs_fork_submit_evaluation(monkeypatch):
+    """Test that forking a process correctly encodes new spans created in each process."""
+    monkeypatch.setenv("_DD_LLMOBS_WRITER_INTERVAL", 5.0)
+    with mock.patch("ddtrace.internal.writer.HTTPWriter._send_payload"):
```

Comment:
The eval metric writer does not extend from `HTTPWriter`, we'll need to use a different mock for this